submission/submission-security/src/main/kotlin/ebi/ac/uk/security/service/SecurityQueryService.kt

```diff
@@ -19,24 +20,28 @@ class SecurityQueryService(
     override fun existsByEmail(
         email: String,
         onlyActive: Boolean,
-    ): Boolean {
```

Comment:
I've been putting these on braces in my last PRs and you've been putting them in single liners 😆 

Do you think we should agree on this? I prefer braces because, in my experience, the debugger acts weirdly in single-liners